app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitors must have a vaccine!")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():

```

Comment:
Also, you don't have to use elif here. Don't use `elif` or `else` after you `raise` an exception or `return` in an if block.